api/src/routes/place.routes.ts

```diff
@@ -7,5 +7,7 @@ placeRoutes.get('/:slug/object_instance',
   (request, response) => placeController.getPlaceObjects(request, response));
 placeRoutes.get('/:slug',
   (request, response) => placeController.getPlace(request, response));
+placeRoutes.get('/:ids/ids',
```

Comment:
@TheMikeSanto Feedback on this endpoint URL. Something about comma seperated ids in a URL makes me feel funny. But maybe that's just me.